dat/Atari - ST.dat

```diff
@@ -1579,6 +1579,12 @@ game (
 )
 
 game (
+	name "Stone Age (Europe) (Amiga + ST)"

```

Comment:
I don't think a separate dat for the private roms would help. Keeping the private roms in doesn't hurt anything.... Having a separate DAT for our customized changes from the upstream, however, would. The [Air Cards (World) change](https://github.com/libretro/libretro-database/pull/27/files#diff-d62c26b1fa607433b57e1c065df51177R41), for instance, would benefit from placing in a different DAT.